### PR TITLE
7.x

### PR DESCRIPTION
**Pull request by MorganDawe**
Corrected an issue with paging which prevent the 'back' button from working, and would not show the correct url (hash) when navigating two page books.

### CODE REVIEW

**Comment by adam-vessey** [comment]
Build failed due to coder issues (actually grabbing output from my local machine, but anyway):

```
[adam@vessey-islandora islandora_internet_archive_bookreader]$ drush coder-review --reviews=production,security,style,i18n,potx,sniffer --no-empty islandora_internet_archive_bookreader
Release standards, Drupal Security Checks, Drupal Coding Standards, Internationalization, Drupal CodeSniffer

sites/vessey-islandora.local.drupal7.test_time/modules/islandora_internet_archive_bookreader/includes/utilities.inc:
 +15: [normal] Remove devel.module debugging function calls.

sites/vessey-islandora.local.drupal7.test_time/modules/islandora_internet_archive_bookreader/theme/theme.inc:
 +39: [minor] Array indentation error, expected 4 spaces but found 6
 +54: [minor] Array indentation error, expected 4 spaces but found 6
 +55: [minor] Array closing indentation error, expected 2 spaces but found 4
 +57: [normal] Remove devel.module debugging function calls.

Status Messages:
 Coder found 1 projects, 11 files, 2 normal warnings, 3 minor warnings, 0 warnings were flagged to be ignored

Coder found 1 projects, 11 files, 2 normal warnings, 3 minor warnings, 0 warnings were flagged to be ignored                                                   [status]
Drush command terminated abnormally due to an unrecoverable error.                                                                                             [error]
[adam@vessey-islandora islandora_internet_archive_bookreader]$
```

**Comment by philsogaDGI** [comment]
- islandora_paged_content provides.
  */
  function islandora_internet_archive_bookreader_remap_highlighting_for_bookreader_results($original_results) {
  -  dsm($original_results);
    $results = array(
    'matches' => array(),
    );

sad panda

**Comment by philsogaDGI** [comment]
There are two dsm() function calls one in the includes/utilities.inc and one in the theme/theme.inc

File: js/islandora_book_reader.js [hunk 1]
```diff
@@ -614,8 +616,11 @@ function IslandoraBookReader(settings) {
     jFullTextDiv.find('.BRfloatMeta').height(600);
     jFullTextDiv.find('.BRfloatMeta').width(600);
     if (1 == this.mode) {
-      var index = this.currentIndex();
-      var pid = this.getPID(index);
+      this.currentIndex();
```
Comment (jonathangreen) [inline js/islandora_book_reader.js]: Does this line do anything anymore?

File: js/islandora_book_reader.js [hunk 2]
```diff
@@ -447,7 +447,7 @@ function IslandoraBookReader(settings) {
       that.search($('#textSrch').val());
       return false;
     });
-
+    this.get
```
Comment (jonathangreen) [inline js/islandora_book_reader.js]: Do we need this for the hack?

**Pull request merged**